backend/controller/ConfigController.ts

```diff
@@ -0,0 +1,78 @@
+import { ConfigKey } from '../../model/configuration/ConfigKey';
+import { DBConfigEntry } from '../../model/configuration/DBConfigEntry';
+import {
+    LoernwerkError,
+    LoernwerkErrorCodes,
+} from '../../model/loernwerkError';
+
+/**
+ * Manages configuration data in the database and handles requests for config requests
+ */
+export class ConfigController {
+    /**
+     * Searches a configurationentry from the database
+     * @param key the configurationkey of the entry
+     * @returns the value of the entry
+     */
+    public static async getConfigEntry(key: ConfigKey): Promise<unknown> {
+        return await DBConfigEntry.findOneBy({ key: key });
```

Comment:
This should check whether the config key exists - even if that's guaranteed by ensureConfig, I feel like this should throw an error if no config entry can be found

Reply:
Also, this should only return the value, not the entire DBConfigEntry